include/bluetooth/services/nus.h

```diff
@@ -86,6 +87,18 @@ int bt_gatt_nus_init(struct bt_gatt_nus_cb *callbacks);
  */
 int bt_gatt_nus_send(struct bt_conn *conn, const u8_t *data, u16_t len);
 
+/**@brief Get maximum data length that can be used for @ref bt_gatt_nus_send.
+ *
+ * @param[in] conn Pointer to connection Object.
+ *
+ * @return Maximum data length.
+ */
+static inline u32_t bt_gatt_nus_max_send(struct bt_conn *conn)
+{
+	/* 3 is for sizeof(struct bt_att_notify) */
+	return bt_gatt_get_mtu(conn) - 3;
```

Comment:
sizeof(struct bt_att_notify)

Reply:
it's part of `att_internal.h`

Reply:
If you want you can refer to 3.4.7.1 Handle Value Notification off the ATT protocol. Maximum supported notification is ATT_MTU - 3

Reply:
Ok anyway - It would be great to add a comment with direct reference to the spec - then this magic number is perfectly fine.

Reply:
done